tests/func/test_live.py

```diff
@@ -255,3 +256,26 @@ def test_dvc_generates_html_during_run(tmp_dir, dvc, mocker, live_stage):
     live_stage(summary=True, live="logs", code=script)
 
     assert show_spy.call_count == 2
+
+
+def test_dvclive_running_subir(tmp_dir, scm, dvc):
+
+    tmp_dir.gen("subdir/train.py", LIVE_SCRIPT)
+    tmp_dir.gen("subdir/params.yaml", "foo: 1")
+    with cd("subdir"):
+        dvc.stage.add(
+            cmd="python train.py",
+            params=["foo"],
+            deps=["train.py"],
+            name="live_stage",
+            live="dvclive",
+            live_no_html=True,
+        )
```

Comment:
You may just be able to set `wdir=subdir` instead of using `cd`. That'll create a `dvc.yaml` file instead of `subdir/dvc.yaml`. Also note that `subdir` should exist before `dvc.stage.add`.

Reply:
I originally writed the test to reproduce the exact workflow the user reported in the bug. But you are right, using `wdir` is enough to cover the case, thanks!